### PR TITLE
Move null check into predicate function

### PR DESCRIPTION
**Pull request by imeredith**
# Description

See [JENKINS-XXXXX](https://issues.jenkins-ci.org/browse/JENKINS-XXXXX).

# Submitter checklist
- [ ] Link to JIRA ticket in description, if appropriate.
- [ ] Change is code complete and matches issue description
- [ ] Appropriate unit or acceptance tests or explanation to why this change has no tests
- [ ] Reviewer's manual test instructions provided in PR description. See Reviewer's first task below.
- [ ] Ran Acceptance Test Harness against PR changes.

# Reviewer checklist
- [ ] Run the changes and verified the change matches the issue description
- [ ] Reviewed the code
- [ ] Verified that the appropriate tests have been written or valid explanation given

@reviewbybees

### CODE REVIEW

File: blueocean-rest-impl/src/main/java/io/jenkins/blueocean/service/embedded/rest/ActionProxiesImpl.java [hunk 1]
```diff
@@ -67,12 +67,12 @@ public Link getLink() {
         for(Action action : Iterables.filter(actions, new Predicate<Action>() {
             @Override
             public boolean apply(Action action) {
+                if(action == null){
+                    return false;
+                }
                 return !BANNED_ACTIONS.contains(action.getClass().getName());
```
Comment (i386) [inline blueocean-rest-impl/src/main/java/io/jenkins/blueocean/service/embedded/rest/ActionProxiesImpl.java]: Simplify to:
`return action != null && !BANNED_ACTIONS.contains(action.getClass().getName());`

**Comment by michaelneale** [comment]
as long as passes automated tests this is a clear :bee:

File: blueocean-rest-impl/src/main/java/io/jenkins/blueocean/service/embedded/rest/ActionProxiesImpl.java (hunk 1 above)
Comment (kzantow) [inline blueocean-rest-impl/src/main/java/io/jenkins/blueocean/service/embedded/rest/ActionProxiesImpl.java]: err... `action == null ||` ?

**Comment by i386** [comment]
LGTM

**Pull request merged**